quickstart.md

```diff
@@ -0,0 +1,186 @@
+might require [additional configuration](https://providers.delta.chat)). Let's
+get username + password from environment variables, configure, and run the
+account:
+
+```python
+import deltachat
+import os
+
+os.getenv("DELTACHAT_DB_PATH", default="/tmp/bot_db/")
+ac = deltachat.account.Account(db_path)
+
+addr = os.getenv("DELTACHAT_BOT_ADDR")
+password = os.getenv("DELTACHAT_BOT_PASSWORD")
+
+ac.run_account(addr=addr, password=password)
+ac.wait_shutdown()
+```
+
+[`account.run_account()`](https://py.delta.chat/api.html#deltachat.account.Account.run_account)
+will run the account, and configure it if necessary (e.g. look up the mail
+servers of your provider).
+
+[`account.wait_shutdown()`](https://py.delta.chat/api.html#deltachat.account.Account.wait_shutdown)
+waits until the account quits running for some reason. This is necessary here
+so the script doesn't quit directly, but stays open as long as the bot is
+running. We want the bot to reply to our messages forever, after all.
+
+## Receiving Messages and Other Events
+
+The account object has an event thread running; if an event happens, it will
+call a hook. You can override this hook in an account plugin with your own
+function to let a bot react to something, e.g. a message.
+
+An account plugin is just a class, which we add to the account object:
+
+```
+class EchoPlugin:
+    pass
+
+ac.add_account_plugin(EchoPlugin)
+```
+
+Now this in itself is not very useful yet. But we can override
+`ac_incoming_message` in the plugin, so we can hook into incoming messages:
+
+```
+class EchoPlugin:
+    @deltachat.account_hookimpl
+    def ac_incoming_message(self, message):
+        print(message.text)
+```
+
+
+
+
+## Sending a Message
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## API
+
+Delta Chat bots use [deltachat-core-rust](https://github.com/deltachat/deltachat-core-rust), the engine doing most of the work.
+
+Your job as bot author is to use the output of that engine, and to give it instructions. You won't parse any email message, check connections, or care for encryption, that's all the engines job.
+
+The core engine can be used natively from rust, and through a [C-API](https://c.delta.chat) from C and C++. (Please note that the rust API is still subject to change, while the C-API is considered stable.)
+
+For the following languages there are bindings to the C-API:
+* Python: [https://py.delta.chat](https://py.delta.chat)
+* NodeJS: [https://github.com/deltachat/deltachat-node](https://github.com/deltachat/deltachat-node)
+* Golang: [https://github.com/hugot/go-deltachat](https://github.com/hugot/go-deltachat)
+
+Additionally there are two small libraries for NodeJS that provide a higher level API to write bots:
+* [deltachat-node-bot-base](https://github.com/deltachat-bot/deltachat-node-bot-base) — for writing bots (duh!),
+* [deltachat-node-webbot-base](https://github.com/deltachat-bot/deltachat-node-webbot-base) — for web-interacting bots (bases on the former).
```

Comment:
don't mention them or update them, they are waay outdated

Reply:
ah yes, everything below `## API` will be removed later, that's just the rest of "development.html" what I copied before committing^^